core/iwasm/compilation/aot_orc_extra.cpp

```diff
@@ -149,10 +182,13 @@ LLVMOrcCreateLLLazyJIT(LLVMOrcLLLazyJITRef *Result,
 
     if (!J) {
         Result = nullptr;
-        return wrap(J.takeError());
+        return NULL;
```

Comment:
`LLVMErrorSuccess` is `0`. `return NULL` will confuse the caller

Reply:
done